### PR TITLE
Theme template for Podcast

### PR DESCRIPTION
**Pull request by Bas-Man**
Proposed Tempate for Podcast
    This is an initial proof of concept.
    Using meta data to create page content.
    
    Adds 'custom_dir' option to mkdocs.yml
    Adds custom directory at the root of the wiki
    Adds podcast.html template file to custom directory
    
    Adds a directory 'podcasts' to docs/
    Adds a sample podcast entry to podcast number 35 '35.md'

### CODE REVIEW

**Comment by WhiteNight121** [comment]
i was trying to get his set up on my side so i could see what it looks like on the site but from what i can see it looks good.

File: docs/podcasts/35.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+  template: podcast.html
+  title: "Episode 35: The Perfect Media Server"
+  number: 35
```
Comment (WhiteNight121) [inline docs/podcasts/35.md:4]: whats the number for? as its already in the title?

Comment (Bas-Man) [reply docs/podcasts/35.md:4]: This is used to create the link to the episode and the chapter links.

Comment (WhiteNight121) [reply docs/podcasts/35.md:4]: hmm ok then that makes sense

**Review by BCNelson — APPROVED** [review]

**Comment by Bas-Man** [comment]
This needs actual feedback on naming and positioning within the wiki.
Also the data currently provided is just filler. Real data needs to be collected/provided.

**Comment by Bas-Man** [comment]
Note:
`- title:` "quotation marks are required if you use punctuation : . in the title."
`   time:` This accepts both integer values or mm:ss format

**Comment by WhiteNight121** [comment]
> 
> 
> This needs actual feedback on naming and positioning within the wiki.
> Also the data currently provided is just filler. Real data needs to be collected/provided.

i think it should go under the wiki folder with a shownotes folder under that and then the template can be there along with with each ep...

**Comment by Bas-Man** [comment]
> > This needs actual feedback on naming and positioning within the wiki.
> > Also the data currently provided is just filler. Real data needs to be collected/provided.
> 
> i think it should go under the wiki folder with a shownotes folder under that and then the template can be there along with with each ep...

The template will need to be kept at the base level. Any files under docs/ will appear on the wiki.

**Pull request closed without merging**